components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtil.java

```diff
@@ -185,6 +186,12 @@ public static void makeAsyncAPICall(SecurityEventTokenPayload securityEventToken
             }
             requestBuilder.setBody(jsonString, ContentType.APPLICATION_JSON);
         } catch (IOException | IdentityEventException | ParseException e) {
+            if(e instanceof IdentityEventException) {
+                if (((IdentityEventException) e).getErrorCode().equals(IO_EXCEPTION_ERROR_CODE)) {
```

Comment:
Don't we need to log the exception here if we are not throwing?

Reply:
It is being logged at lower level: https://github.com/wso2-extensions/identity-outbound-adapters/blob/main/components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/EventPayloadCryptographyUtils.java#L203